compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -221,18 +220,19 @@ function finishCallStmt(Tokenizer tok, CallStmt expr, Position startPos) returns
 function finishCheckingCallStmt(Tokenizer tok, CheckingKeyword checkingKeyword, Position startPos) returns CallStmt|err:Syntax {
     Token? t = tok.current();
     if t is "check"|"checkpanic" {
+        Position operandStartPos = tok.currentStartPos();
```

Comment:
Don't understand what's going on here. The current token start position is the keyword position, but you're getting the keyword positionm from somewhere else.

Reply:
this function is called after ingesting `CheckingKeyword` here we are dealing with the case like `check check f()` and `operandStartPos` is the `startPos` of second `check`